bika/lims/content/bikasetup.py

```diff
@@ -293,44 +291,6 @@ def getCounterTypes(self, instance=None):
                 "will be calculated and plotted"),
         )
     ),
-    BooleanField(
-        'IncludePreviousFromBatch',
-        schemata="Results Reports",
-        default=False,
-        widget=BooleanWidget(
-            label=_("Include Previous Results From Batch"),
-            description=_(
-                "If there are previous results for a service in the "
-                "same batch of Samples, they will be displayed "
-                "in the report.")
-        )
-    ),
-    IntegerField(
-        'BatchEmail',
-        schemata="Results Reports",
-        required=1,
-        default=5,
-        widget=IntegerWidget(
-            label=_("Maximum columns per results email"),
-            description=_(
-                "Set the maximum number of samples per results email. "
-                "Too many columns per email are difficult to read for some clients "
-                "who prefer fewer results per email"),
-        )
-    ),
-    TextField(
-        'ResultFooter',
```

Comment:
Removal of "ResultFooter" field causes the following traceback, because of https://github.com/senaite/senaite.impress/blob/senaite-impress-core-integration/src/senaite/impress/analysisrequest/reportview.py#L126 (from `senaite.impress:senaite-impress-core-integration` branch )

```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module senaite.impress.decorators, line 41, in decorator
  Module senaite.impress.ajax, line 65, in __call__
  Module senaite.impress.ajax, line 176, in ajax_render_reports
  Module senaite.impress.publishview, line 189, in render_multi_report
  Module senaite.impress.publishview, line 306, in read_template
  Module Products.Five.browser.pagetemplatefile, line 59, in __call__
  Module zope.pagetemplate.pagetemplate, line 132, in pt_render
  Module five.pt.engine, line 98, in __call__
  Module z3c.pt.pagetemplate, line 158, in render
  Module chameleon.zpt.template, line 297, in render
  Module chameleon.template, line 191, in render
  Module chameleon.template, line 171, in render
  Module ff3aba67e1661d0128c35d155633ebdd.py, line 269, in render
  Module senaite.impress.analysisrequest.reportview, line 126, in get_footer_text
  Module senaite.core.supermodel.model, line 105, in __getattr__
AttributeError: getResultFooter
```

Reply:
Sorry, I accidently lost some code. 
The new Impress Controlpanel should look now like this and provide the footer settings:

<img width="940" alt="" src="https://user-images.githubusercontent.com/713193/52236436-2708c580-28c7-11e9-88de-8466170ec5a2.png">